internal/ini/walker_test.go

```diff
@@ -13,7 +11,7 @@ import (
 
 func TestValidDataFiles(t *testing.T) {
 	const expectedFileSuffix = "_expected"
-	filepath.Walk("./testdata/valid", func(path string, info os.FileInfo, err error) error {
+	err := filepath.Walk("./testdata/valid", func(path string, info os.FileInfo, err error) error {
```

Comment:
Was already existing before, but this should use `filepath.Join("testdata", "valid")` instead of the literal string. `filepath.Join` will correctly join the path for the OS.